app/models/node.rb

```diff
@@ -920,8 +924,14 @@ def self.research_notes
                .group('node.nid')
                .collect(&:nid)
 
-    Node.where(type: 'note')
-                .where('node.nid NOT IN (?)', nids)
+    notes = Node.where(type: 'note')
+
+    # if the nids are empty, it should not be used in the query
+    if nids.empty?
+      notes
+    else
+      notes.where('node.nid NOT IN (?)', nids)
+    end
```

Comment:
`notes.where('node.nid NOT IN (?)', nids)` ?

Reply:
thanks, see comment [here](https://github.com/publiclab/plots2/pull/8800/files/c5954b67e9b5c1e72d6e6192a73883f167362c87#r540164441) on the same